plugin/trino-mysql/src/test/java/io/trino/plugin/mysql/TestingMySqlServer.java

```diff
@@ -30,7 +30,7 @@
 public class TestingMySqlServer
         implements AutoCloseable
 {
-    public static final String DEFAULT_IMAGE = "mysql:8.0.12";
+    public static final String DEFAULT_IMAGE = "mysql:8.0.26";
```

Comment:
Any reason you specified `8.0.26` instead of `8.0.29`?

Reply:
No specified reason, Just tried a image to fix the mac issue. 8.0.29 is the latest one, I think we can use the latest one. Thanks for the question!